File: android/farmerchat-sdk/src/main/java/com/farmerchat/sdk/domain/model/guest/InitializeGuestUserResponse.kt
package com.farmerchat.sdk.domain.model.guest

internal data class InitializeGuestUserResponse(
    val access_token: String,
    val refresh_token: String,
    val user_id: String?,
    val created_now: Boolean? = null,   // true = new user, false = existing user returned
    val show_crops_livestocks: Boolean = false,
    val country_code: String? = null,
    val country: String? = null,
    val state: String? = null
)

File: ios/Sources/FarmerChatSDK/Models/Response/InitializeGuestUserResponse.swift
import Foundation

struct InitializeGuestUserResponse: Decodable {
    let access_token: String
    let refresh_token: String
    let user_id: String?
    let created_now: Bool?   // true = new user, false = existing user returned for this device_id
    let country_code: String?
    let country: String?
    let state: String?
}
